<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>单位转换工具</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout_3">
    <item>
     <widget class="QComboBox" name="comboBox_unit">
      <item>
       <property name="text">
        <string>长度</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>角度</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>面积</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>速度</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>温度</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>压强</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>体积</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>时间</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>位置</string>
       </property>
      </item>
     </widget>
    </item>
    <item>
     <widget class="QGroupBox" name="groupBox">
      <property name="title">
       <string>输入</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QLineEdit" name="lineEdit_input"/>
         </item>
         <item>
          <widget class="QComboBox" name="comboBox_input"/>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QGroupBox" name="groupBox_2">
      <property name="title">
       <string>输出</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QLineEdit" name="lineEdit_outPut"/>
         </item>
         <item>
          <widget class="QComboBox" name="comboBox_outPut"/>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>400</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
